app/src/main/java/app/insti/adapter/CommentRecyclerViewAdapter.java

```diff
@@ -0,0 +1,186 @@
+package app.insti.adapter;
+
+import android.app.Activity;
+import android.content.ClipData;
+import android.content.ClipboardManager;
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.CardView;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.PopupMenu;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.squareup.picasso.Picasso;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import app.insti.R;
+import app.insti.activity.MainActivity;
+import app.insti.api.RetrofitInterface;
+import app.insti.api.model.Venter;
+import app.insti.utils.DateTimeUtil;
+import de.hdodenhof.circleimageview.CircleImageView;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+/**
+ * Created by Shivam Sharma on 23-09-2018.
+ */
+
+public class CommentRecyclerViewAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+
+    private static final String TAG = CommentRecyclerViewAdapter.class.getSimpleName();
+
+    private Context context;
+    private LayoutInflater inflater;
+    private String sessionId, userId;
+    private Activity activity;
+
+    private List<Venter.Comment> commentList = new ArrayList<>();
+
+    public CommentRecyclerViewAdapter(Activity activity, Context context, String sessionId, String userId) {
+        this.context = context;
+        this.sessionId = sessionId;
+        this.userId = userId;
+        inflater = LayoutInflater.from(context);
+        this.activity = activity;
+    }
+
+    public class CommentsViewHolder extends RecyclerView.ViewHolder {
+
+        private CardView cardView;
+        private CircleImageView circleImageView;
+        private TextView textViewName;
+        private TextView textViewCommentTime;
+        private TextView textViewComment;
+        final RetrofitInterface retrofitInterface = activity != null ? ((MainActivity) activity).getRetrofitInterface() : null;
```

Comment:
MainActivity$getRetrofitInterface won't exist after #218

Reply:
What is the solution for this?

Reply:
Use `Utils$getRetrofitInterface` instead